Sources/TuistAutomation/Utilities/XcodeBuildDestination+Find.swift

```diff
@@ -0,0 +1,55 @@
+import TSCBasic
+import TSCUtility
+import TuistCore
+import TuistGraph
+import TuistSupport
+
+public extension XcodeBuildDestination {
```

Comment:
Do we want to use this extension outside `TuistAutomation`?

I would also love to see some tests for this – especially, if we do want to use it outside of this module.